### DISCUSSION TITLE
Determine appropriate playbook categories

### DISCUSSION THREAD

**Discussion by moin-oss** [Green-Software-Foundation/green-software-playbooks#10] [General]
Let us update the categories we currently have here https://github.com/Green-Software-Foundation/green-software-playbooks/blob/dev/docs/playbook.md to something more useful.  Initially, we considered using the following:

- Software as a Service (SaaS)
  - Webapps
  - Daemons
  - Data Science Applications
- Infrastructure as a Service (IaaS)
  - CI/CD
  - Caches
  - Task Management
  - Databases
  - Event Streaming

However, from the standpoint of an engineering team or individual contributor, these might make it more difficult to navigate to the appropriate playbooks.  Given that contributors are organized by function, it probably makes more sense to categorize them as such, for example:

- App developer
- Data Engineering/Data Science
- DevOps/SRE
- UX/UI Design

Let's try to specify our playbook categories.

**Comment by f-mellinghoff** [comment]
Some additional suggestions: 
- Scrum Master / PO / Project Manager
- Architect
- Software Engineer
- End User
